### DISCUSSION TITLE
blog/posts/interviews/ringcentral-campus-recruitment-2024/

### DISCUSSION THREAD

**Discussion by giscus** [KTS-o7/KTS-o7.github.io#7] [General]
# blog/posts/interviews/ringcentral-campus-recruitment-2024/

Ringcentral Interview Experience for SDE-1 (Oncampus) Eligibility Criteria 8.5+ CGPA Ciruit Branches No backlogs Batch of 2025 Qualification Round This was an online test conducted on HackerRank. The test was of 90 minutes and had 2 sections:
Section 1: MCQs This section had 5 MCQs. Got questions from Huffman encoding, Hamming code, Time complexity analysis. There was a positive score for correct answers and no negative marking for wrong answers. Section 2: Coding This section had 2 coding questions: It was Leetcode hard-medium level questions.

https://kts-o7.github.io/blog/posts/interviews/ringcentral-campus-recruitment-2024/

**Comment by gururajbg** [comment]
Congratulations 🎉🎉. And thank you so much for sharing your experience.

**Reply by KTS-o7** [reply]
Thank you ! In coming days will be adding resources for preparation so that juniors can have some direction !

**Reply by gururajbg** [reply]
That is very kind of you , thank you
